behave/run_test_local_chrome_driver.sh

```diff
@@ -1,7 +1,6 @@
 export BROWSER=chrome
 #these directories need to be overridden as they do not get picked up from cla-end-to-end-tests
 export ARTIFACTS_DIRECTORY="."
```

Comment:
you will need to have some sort of directory in here I think or this won't work

Reply:
think will need `export DATA_DIRECTORY="/tmp"` to run properly. Also, is `ARTIFACTS_DIRECTORY` required?

Reply:
I have removed the `ARTIFACTS_DIRECTORY` setting